### DISCUSSION TITLE
Reverse proxy for Data Importer?

### DISCUSSION THREAD

**Discussion by chriscn** [firefly-iii/firefly-iii#7365] [Firefly III]
I'm using LinuxServer's SWAG and can't get the data importer to play nicely with my reverse proxy. I'm using the same template for both firefly and the importer.

Just getting 502 errors and I've ensured that the trusted proxies is set to `**`

**Comment by JC5** [comment]
Are the port settings correct? Try `*` (single asterisk)

**Reply by chriscn** [reply]
This is what I've tried and unfortunately no one of them are working:
- Importer works when using a Personal Access Token and then connecting directly
- Seems to be related to redirect loop at `/token` 502 Bad Gateway

**Reply by treed593** [reply]
I am seeing the same thing

**Reply by JC5** [reply]
What's the URL when it ends up with the 502?

**Comment by kevindurb** [comment]
I also get redirect loop at /token with 502s when i use a reverse proxy...
